pkg/apis/resource/types.go

```diff
@@ -251,9 +250,9 @@ const ResourceSliceMaxDeviceCounterConsumptions = 32
 const ResourceSliceMaxSharedCountersCounters = 32
```

Comment:
Do we still need this?

Reply:
There were a few constants that I added back by mistake. I've cleaned that up now.